lib/setup.py

```diff
@@ -1,11 +1,22 @@
+import platform
 import setuptools
+import subprocess
 
 from pipenv.project import Project
 from pipenv.utils import convert_deps_to_pip
 
 pfile = Project(chdir=False).parsed_pipfile
 requirements = convert_deps_to_pip(pfile['packages'], r=False)
 
+# Check whether xcode tools are available before making watchdog a
+# dependency (only if the current system is a Mac).
+if (platform.system() == 'Darwin' and
```

Comment:
Maybe we should pull this into `util.py`, since it seems we'll need to do this check in a few different places.